content/rs/administering/designing-production/hardware-requirements.md

```diff
@@ -31,9 +31,9 @@ requirements for a production system:
 
 | Item | Description | Minimum Requirements | Recommended |
 |------------|-----------------|------------|-----------------|
-| # of nodes per cluster  | At least 3 nodes are needed in order to support a reliable, highly available deployment that handles process failure, node failure, and network split events in a consistent manner. | 3 nodes | >=3 nodes; must be odd number of nodes to maintain quorum |
+| # of nodes per cluster  | At least 3 nodes (2 data nodes and 1 quorum node) are needed in order to support a reliable, highly available deployment that handles process failure, node failure, and network split events in a consistent manner. | 3 nodes | >=3 nodes; must be odd number of nodes to maintain quorum |
```

Comment:
We don't need to mention "quorum node" here, as it's not a requirement to have a quorum node, and the cluster can have 3 "regular" nodes